developers/weaviate/current/configuration/authentication.md

```diff
@@ -107,33 +105,33 @@ authentication:
       # where an example realm 'my-weaviate-usecase' was created. The exact
       # path structure will depend on the token issuer of your choice. Please
       # see the respective documentation of your issuer about which endpoint
-      # implements OIDC Discovery.
-      issuer: http://my-token-issuer/auth/realms/my-weaviate-usecase
-
-      # username_claim (required) tells weaviate which claim to use for extracting
-      # the username. The username will be passed to the authorization module.
-      username_claim: email
-
-      # groups_claim (optional) tells weaviate which claim to use for extracting
-      # the groups. Groups must be an array of string. If groups_claim is not set
-      # weaviate will not try to extract groups and pass an empty array to the 
-      # authorization module.
-      groups_claim: groups
+      # implements OIDC Discovery.      
+      AUTHENTICATION_OIDC_ISSUER: 'http://my-token-issuer/auth/realms/my-weaviate-usecase'
 
       # client_id (required unless skip_client_id_check is set to true) tells 
       # weaviate to check for a particular OAuth 2.0 client_id in the audience claim.
       # This is to prevent that a token which was signed by the correct issuer
       # but never intended to be used with weaviate can be used for authentication.
       #
       # For more information on what clients are in OAuth 2.0, see
-      # https://tools.ietf.org/html/rfc6749#section-1.1
-      client_id: my-weaviate-client
+      # https://tools.ietf.org/html/rfc6749#section-1.1      
+      AUTHENTICATION_OIDC_CLIENT_ID: 'my-weaviate-client'
+
+      # username_claim (required) tells weaviate which claim to use for extracting
+      # the username. The username will be passed to the authorization module.      
+      AUTHENTICATION_OIDC_USERNAME_CLAIM: 'email'
+
+      # groups_claim (optional) tells weaviate which claim to use for extracting
+      # the groups. Groups must be an array of string. If groups_claim is not set
+      # weaviate will not try to extract groups and pass an empty array to the 
+      # authorization module.      
+      AUTHENTICATION_OIDC_GROUPS_CLAIM: 'groups'
```

Comment:
afaik the groups do not really work. So maybe we should remove this for now? @byronvoorbach

Reply:
Discussed with @dirkkul and agreed to merge as is for now as it is consistent with the existing wording. Let's remove this later if necessary.